components/ide/jetbrains/backend-plugin/src/main/resources-latest/META-INF/extensions.xml

```diff
@@ -7,5 +7,7 @@
     <extensions defaultExtensionNs="com.intellij">
         <projectService serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodTerminalService" client="guest" preload="true"/>
         <projectService serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodPortForwardingService" preload="true"/>
+        <gateway.customization.performance implementation="io.gitpod.jetbrains.remote.latest.GitpodMetricControlProvider"/>
```

Comment:
i think it would make sense to put them before standard controllers, there should be an attribute to specify before in plugin.xml